lib/resultsprocessor.rb

```diff
@@ -1,5 +1,6 @@
 # encoding: UTF-8 
 #
+require 'codemessage'
```

Comment:
Unit testing results processor revealed that codemessage is implicitly required here in resultsprocessor.  It needed to be added explicitly when testing resultsprocessor in an isolated fashion.